data/nav.yml

```diff
@@ -8,6 +8,14 @@
       path: "tutorials/getting-started"
     - name: "Glossary"
       path: "pipelines/glossary"
+    - name: Migration
+      children:
+        - name: Overview
+          path: pipelines/migration-overview
+        - name: From Jenkins
+          path: pipelines/migrate-from-jenkins
```

Comment:
(**non-blocking, question**) I we're planning on adding more guides would it be worth using a route that roughly matches our navigation (ie. `pipelines/migration/jenkins`)?

Reply:
Until we do some proper IA work, I want to keep any new URLs flexible + limit the nesting from getting too deep. Keeping it at `pipelines/` allows us to move it anywhere in the navigation under Pipelines without worrying about the URL not matching.